invenio_previewer/__init__.py

```diff
@@ -185,21 +184,25 @@
 Previewing a file
 ~~~~~~~~~~~~~~~~~
 
-We should be able to see now the result HTML generated from the markdown file:
+Let's try to preview the MarkDown file. What we expect is that the previewer
+will be able to convert the MarkDown to HTML on the fly and we will see a nice
+MarkDown preview:
 
 >>> with app.test_client() as client:
 ...     res = client.get('/records/1/preview/markdown.md')
 
-A lot is happening here so let's take it in steps:
+Here a more detailed explanation:
 
-1. Records-UI resolves ``/records/1/preview/markdown.md`` into a record and
-   persistent identifier which is then passed to Invenio-Previewer.
-2. Invenio-Previewer looks to the ``files`` key in the record and expect to
-   find a list of dictionaries, with each dictionary representing a file
-   (note: by default the first file in the list will be previewed).
-3. Invenio-Previewer determines the file extensions and searchers for a
-   preview plugin to handle the request.
-4. The preview plugin now finally takes care of all rendering.
+1. When calling the url ``/records/1/preview/markdown.md``, Invenio-Records-UI
+   resolves the record from the PID 1 and it passes it to Invenio-Previewer.
+2. Invenio-Previewer retrieves the file object by the filename ``markdown.md``
+   using the ``invenio_previewer.ext.record_file_factory`` property
+   (implemented by
+   `Invenio-Records-Files <https://invenio-records-files.rtfd.io>`_ if
+   installed).
+3. With the file object, Invenio-Previewer iterates in the list of configured
```

Comment:
I am not sure `in` adverb could be used in this context. For me more natural sounds`iterates trough` or `iterates over` or just `iterates the list`. I am might be wrong, just a thought.

Reply:
For me iterates over sounds best

Reply:
Your English is much better than mine guys, I will change it thanks!

Reply:
@ntarocco I already did :)